src/platform/zcu102_revmin/zcu102_revmin_pfm.tcl

```diff
@@ -1,8 +1,8 @@
-platform -name zcu102_revmin -desc "A basic platform targeting the ZCU102 evaluation board, which includes 4GB of DDR4 for the Processing System, 512MB of DDR4 for the Programmable Logic, 2x64MB Quad-SPI Flash and an SDIO card interface. More information at https://www.xilinx.com/products/boards-and-kits/ek-u1-zcu102-g.html" -hw ./zcu102_revmin.xsa -out ./output  -prebuilt
+platform -name zcu102_revmin -desc "A basic platform targeting the ZCU102 evaluation board, which includes 4GB of DDR4 for the Processing System, 512MB of DDR4 for the Programmable Logic, 2x64MB Quad-SPI Flash and an SDIO card interface. More information at https://www.xilinx.com/products/boards-and-kits/ek-u1-zcu102-g.html" -hw ./zcu102_revmin.xsa -out ./outpu
```

Comment:
*output?

Reply:
Hi Min,
Sorry, it was a typo. correcting it.